apps/checkout-app/pages/customization.tsx

```diff
@@ -24,6 +26,7 @@ const Customization = () => {
     metadataQuery.data?.app?.privateMetadata || []
   );
   const customizationSettings = getCustomizationSettings(
```

Comment:
Just wondering, why pass intl all the 4 levels down there instead of making this a hook like `const getCustomizationSettings = useGetCustomizationSettings()` ? I think if there is a need to pass props that deep then this should be a hook